runconfig/parse_test.go

```diff
@@ -157,9 +161,6 @@ func TestParseRunVolumes(t *testing.T) {
 	if _, _, err := parse(t, "-v /tmp:"); err == nil {
 		t.Fatalf("Error parsing volume flags, `-v /tmp:` should fail but didn't")
 	}
-	if _, _, err := parse(t, "-v /tmp:ro"); err == nil {
```

Comment:
Meaning, now it is supposed to work with this syntax and not fail like on this test :wink:.